### DISCUSSION TITLE
Spinal Lances not appearing

### DISCUSSION THREAD

**Discussion by SMG001** [bcholmes/StarTrek2d20#130] [Q&A] [answered]
I don't know if it's a bug or if it's just me being dumb, but the Spinal Lances aren't appearing in the ship's talent list, so what's going on?

**Comment by Felderburg** [comment, marked as answer]
In the UP book pg 78, they're an energy weapon system, not a talent (so a modifier like arrays, banks, and cannons). They're also limited to 25th-century ships (or GM permission). It looks like in the app, they're only available with a custom spaceframe or simplified build? I haven't used ships that much, but I just tried making a 2401 galaxy class, and there wasn't an option to modify the default weapons it came with.

**Reply by SMG001** [reply]
I see. Thank you for clearing that up

**Reply by Felderburg** [reply]
Well, it's a good thing to ask. I do think that we should be able to change the weapons a ship has when using a standard spaceframe, which I didn't realize we can't do.

**Reply by TakeshiYamato01** [reply]
That is what the Expanded Munitions Talent is meant for, adding another weapons system to the spaceframe. If we take that talent, we should be able to add another weapon.
